includes/admin/class-sensei-wccom-connect-notice.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * File containing the class Sensei_WCCOM_Connect_Notice
+ *
+ * @package sensei-lms
+ * @since   3.11.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class that handles showing the WooCommerce.com connect notice.
+ *
+ * @access private
+ *
+ * @class Sensei_WCCOM_Connect_Notice
+ */
+class Sensei_WCCOM_Connect_Notice {
+	const DISMISS_NOTICE_NONCE_ACTION   = 'sensei-lms-cancelled-wccom-connect-dismiss';
+	const DISMISSED_NOTIFICATION_OPTION = 'sensei-cancelled-wccom-connect-dismissed';
```

Comment:
I think there is an extra `cancelled` in there.

Reply:
Removed in 2efae3723f5e17c807d39918eff6e2a3ffc33256